### PR TITLE
Maintenance week updates

### PR DESCRIPTION
**Pull request by ehanson8**
### Purpose and background context
Updates app according to our [maintenance week documentation](https://mitlibraries.atlassian.net/wiki/spaces/IN/pages/3445948428/DataEng+Application+Maintenance)

### How can a reviewer manually see the effects of these changes?
Given that CloudConnector is required to connect to the Data Warehouse and only available on `stage` and `prod`, there is no easy way to manually test these changes aside from verifying that `make test` and `make lint` still work.

### Includes new or updated dependencies?
YES

### Changes expectations for external applications?
NO

### Developer
- [x] All new ENV is documented in README
- [x] All new ENV has been added to staging and production environments
- [x] All related Jira tickets are linked in commit message(s)
- [x] Stakeholder approval has been confirmed (or is not needed)

### Code Reviewer(s)
- [ ] The commit message is clear and follows our guidelines (not just this PR message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The provided documentation is sufficient for understanding any new functionality introduced
- [ ] Any manual tests have been performed and verified
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: patronload/email.py [hunk 1]
```diff
@@ -44,19 +39,8 @@ def populate(  # noqa pylint R0913 too many arguments
...
-            self["Bcc"] = bcc
-        if reply_to:
-            self["Reply-To"] = reply_to
         if body:
             self.set_content(body)
-        if attachments:
-            for attachment in attachments:
-                self.add_attachment(
-                    attachment["content"], filename=attachment["filename"]
-                )
```
Comment (ghukill) [inline patronload/email.py:59]: Was it determined that we don't need cc, bcc, reply_to, and attachments?

File: patronload/patron.py [hunk 2]
```diff
@@ -19,83 +19,69 @@
...
-    patron_dict: dict[str, Any],
-    six_months: str,
-    two_years: str,
-) -> BeautifulSoup:
-    """
-    Populate the fields common to both staff and student patron records.
-
-    The order of the patron record fields must be in same order as they in the
-    STAFF_FIELDS and STUDENT_FIELDS value lists.
+def patrons_xml_string_from_records(
```
Comment (ghukill) [inline patronload/patron.py:22]: Not a comment on the maintenance work -- this refactor and reordering makes sense! -- but it did take me a bit to understand from the diff (had to compare two branches by toggling between them) that this was just ordering, and not a meaningful change to code.

**Review by ghukill — APPROVED** [review]
Looks good!  I left a comment about what appers to be the dropping of `cc, bcc, reply_to, etc.`, but other than that, looks like good updates.

File: patronload/patron.py (hunk 2 above)
Comment (ehanson8) [reply patronload/patron.py:22]: Sorry that's why I mentioned it in the commit message but in retrospect, it should have been a separate commit

File: patronload/email.py (hunk 1 above)
Comment (ehanson8) [reply patronload/email.py:59]: This is just for sending the logs and none of those fields are used. I believe I stole this from another repo without any modifications and I don't see a use case where this app would start using those fields

File: patronload/patron.py (hunk 2 above)
Comment (ghukill) [reply patronload/patron.py:22]: Apologies are mine!  I missed that in the commit message; think that's more than sufficient.

**Review by jonavellecuerdo — APPROVED** [review]

**Pull request merged**